buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -75,10 +90,23 @@ class AirbytePythonPlugin implements Plugin<Project> {
         def extension = project.extensions.create('airbytePython', AirbytePythonConfiguration)
 
         def venvDirectoryName = '.venv'
+
+        // Add a task that allows cleaning up venvs to every python project
+        project.task('cleanPythonVenv', type: Exec) {
+            commandLine 'rm'
+            args '-rf', "$project.projectDir.absolutePath/$venvDirectoryName"
+        }
+        project.clean.dependsOn project.cleanPythonVenv
+
         project.plugins.apply 'ru.vyarus.use-python'
 
+        // Configure gradle python plugin.
         project.python {
             envPath = venvDirectoryName
+            if (Helpers.isPyenvInPath()) {
+                // When using pyenv, prefer 'python' instead of 'python3'.
+                pythonBinary = 'python'
+            }
```

Comment:
This takes care of https://airbytehq-team.slack.com/archives/C02U2SSHP9S/p1669756536254519
cc @erohmensing @rodireich

Reply:
Nice.

Reply:
Nice!